rspirv/lift/mod.rs

```diff
@@ -24,11 +24,18 @@ use spirv;
 /// Reverse lookup table that associates SPIR-V <id> with SR tokens.
 type LookupMap<T> = HashMap<spirv::Word, Token<T>, BuildHasherDefault<FxHasher>>;
 
+struct OpInfo {
```

Comment:
This may be clarified in a later commit but it isn't at all obvious what this is doing.

Reply:
will do